trie/proof_test.go

```diff
@@ -208,7 +208,7 @@ func TestRangeProof(t *testing.T) {
 	}
 }
 
-// TestRangeProof tests normal range proof with two non-existent proofs.
+// TestRangeProofWithNonExistentProof tests normal range proof with two non-existent proofs.
```

Comment:
Please revert, this has been already fixed upstream in [geth](https://github.com/ethereum/go-ethereum/blob/ed1d46b3d31f250e5385a4b8b8f97293acf8adee/trie/proof_test.go#L170) 🙏